src/services/AppUserService.ts

```diff
@@ -119,6 +119,14 @@ export class AppUserService {
 
     return false;
   }
+
+  isGuest(appUser: AppUser): boolean {
+    if (appUser.role !== AppUserRole.GUEST) {
```

Comment:
`return appUser.role === AppUserRole.GUEST`

Reply:
I keep forgetting about this lol...